python/neuroglancer/pipeline/storage.py

```diff
@@ -107,6 +117,50 @@ def get_file(self, file_path):
             content = self._maybe_uncompress(content)
         return content
 
+    def get_files(self, file_paths):
+        """
+        returns a list of files faster by using threads
+        """
+
+        file_paths = deque(file_paths)
+
+        results = []
```

Comment:
The other upside to using a generator is that you can start using results immediately.